src/content/docs/en/pages/store-journey/sql/delete-record.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: How to delete a record in Edge SQL database
+description: >-
+  Discover step-by-step instructions to delete a record in an Edge SQL database for efficient data management.
+permalink: /documentation/products/store/sql/delete-record/
+meta_tags: >-
+  Edge SQL, SQL database, delete record, database management, CRUD operations, API integration, data deletion, SQL tutorial, Edge SQL guide, database modification, SQL commands
+namespace: docs_store_journey_sql_delete_record
+menu_namespace: storeMenu
+---
+
+This guide demonstrates how to use Azion **Edge SQL** to delete a record in your database using the Azion API.
+
+## Deleting a record
+
+Run the following `POST` request in your terminal, replacing `[TOKEN VALUE]` with your [personal token](/en/documentation/products/guides/personal-tokens/) and `{id_database}` with the ID of the database you want to create a table into:
+
+```bash
+curl --location --request POST 'https://api.azion.com/v4/edge_sql/databases/{id_database}/query' \
+--header 'Authorization: Token <token_value>' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+"statements": [
+"delete from people where name = '\''fulaninho'\'' "
```

Comment:
Perhaps instead of 'people' there could be 'users' like in other examples and the table created in the "Create Table" guide.